packages/gatsby/cache-dir/__tests__/loader.js

```diff
@@ -305,9 +306,16 @@ describe(`Production loader`, () => {
         })
       )
 
-      const expectation = await prodLoader.loadPage(`/mypage/`)
+      const expectation = await prodLoader.loadPage(`/mypage`)
```

Comment:
Why trailing slash is removed here? It doesn't pass otherwise? If so, that's a problem ;)

Reply:
I just removed for consistency 😂 It passes either way since we get normalised path from raw path